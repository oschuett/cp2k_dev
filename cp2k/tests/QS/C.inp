&CP2K
 PROGRAM      Quickstep
 IOLEVEL      3
 FFTLIB       FFTSG
&END

&DFT
 FUNCTIONAL   Pade
&END

&QS
 NGRID        1
 CUTOFF       240
 REL_CUTOFF   30
 EPS_RHO      1.0E-10
 EPS_GVG      1.0E-8
&END

&SCF
 GUESS        ATOMIC
 EPS_SCF      1.0E-5
 MAX_SCF      30
 MIXING       0.4
 SMEAR        0.02
 ADDED_MOS    4 0
&END

&CELL
 UNIT       ANGSTROM
 ABC        12.0 12.0 12.0
&END

&PRINT LOW
 BASIS_SET
 POTENTIAL
 MO_OCCUPATION_NUMBERS
&END

&KIND C
 BASIS_SET DZVP-GTH-PADE
 POTENTIAL GTH-PADE-q4
&END

&COORD
 C   0.000000  0.000000  0.000000
&END
