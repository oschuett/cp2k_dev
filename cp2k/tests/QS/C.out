 
  **** **** ******  **  PROGRAM STARTED AT              Thu Apr 18 10:35:02 2002
 ***** ** ***  *** **   PROGRAM STARTED ON                                lilith
 **    ****   ******    PROGRAM STARTED BY                                 krack
 ***** **    ** ** **   PROGRAM PROCESS ID                                105472
  **** **  *******  **  PROGRAM STARTED IN   /gpfs/home/krack/work/cp2k/tests/QS
 
 CP2K| Program name                                                           QS
 CP2K| Input file name                                                     C.inp
 CP2K| Project name                                                      project
 CP2K| Default FFT library                                                 FFTSG
 CP2K| Default basis set file                                          BASIS_SET
 CP2K| Default external potential file                                 POTENTIAL
 CP2K| Default library path                                                   ./
 CP2K| Global print level                                                      3
 CP2K| Total number of processors                                              1
 CP2K| This output from processor                                              0
 
 ******************************************************************************
 ******************************************************************************
 **                                                                          **
 **     #####                         ##              ##                     **
 **    ##   ##            ##          ##              ##                     **
 **   ##     ##                       ##            ######                   **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####   **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##  **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######   **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##       **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##       **
 **           ##                                                    ##       **
 **                                                                          **
 **                                                ... make the atoms dance  **
 **                                                                          **
 **                         Version 3.0 (June 2001)                          **
 **                                                                          **
 **     Copyright (C) by MPI fuer Festkoerperforschung, Stuttgart (2000)     **
 **                                                                          **
 ******************************************************************************
 ******************************************************************************

 DFT| Exchange-correlation functional                                      Pade
 DFT| Sets of spin states:                                                    1
 DFT| Cutoffs: density                                              0.10000E-09
 DFT|          gradient                                             0.10000E-07
 DFT| XC mesh (PW density)                                              default

 QS| Method:                                                                GPW
 QS| Number of grid levels:                                                   1
 QS| Density cutoff [a.u.]:                                               120.0
 QS| Grid level progression factor:                                         2.0
 QS| Relative density cutoff [a.u.]:                                       30.0
 QS| Interaction thresholds: eps_pgf_orb:                               1.0E-06
 QS|                         eps_core_charge:                           1.0E-12
 QS|                         eps_rho_gspace:                            1.0E-10
 QS|                         eps_rho_rspace:                            1.0E-10
 QS|                         eps_gvg_rspace:                            1.0E-08
 QS|                         eps_ppl:                                   1.0E-12
 QS|                         eps_ppnl:                                  1.0E-12

 CELL| Volume [ANGSTROM**3]:                                           1728.000
 CELL| Vector a [ANGSTROM]:      12.000     0.000     0.000    |a| =     12.000
 CELL| Vector b [ANGSTROM]:       0.000    12.000     0.000    |b| =     12.000
 CELL| Vector c [ANGSTROM]:       0.000     0.000    12.000    |c| =     12.000
 CELL| Angle (b,c) [degree]:                                             90.000
 CELL| Angle (a,c) [degree]:                                             90.000
 CELL| Angle (a,b) [degree]:                                             90.000
 CELL| Grid size for subcell generation                                       2


 BASIS SET INFORMATION (Unnormalised Gaussian-type functions)

  1. Atomic kind: C

   - GTO basis set information

     GTO basis set name: DZVP-GTH-PADE

     Set   Shell     n   l            Exponent    Coefficient

       1       1     2   0            4.336251       0.149079
                                      1.288182      -0.029263
                                      0.403776      -0.688204
                                      0.118788      -0.396442
       1       2     3   0            4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       1.000000
       1       3     3   1            4.336251      -0.087812
                                      1.288182      -0.277557
                                      0.403776      -0.471229
                                      0.118788      -0.405804
       1       4     4   1            4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       1.000000

       2       1     3   2            0.600000       1.000000


 ATOMIC KIND INFORMATION

  1. Atomic kind: C

     Number of atoms of this atomic kind in the cell:      1

   - Orbital basis set information

     Orbital basis set name: DZVP-GTH-PADE

     Number of orbital shell sets:                 2
     Number of orbital shells:                     5
     Number of primitive Cartesian functions:      5
     Number of Cartesian basis functions:         14
     Number of spherical basis functions:         13

     Normalised Cartesian orbitals

     Set   Shell   Orbital            Exponent    Coefficient

       1       1    2s                4.336251       0.319274
                                      1.288182      -0.025218
                                      0.403776      -0.248447
                                      0.118788      -0.057170

       1       2    3s                4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       0.144208

       1       3    3px               4.336251      -0.783228
                                      1.288182      -0.542955
                                      0.403776      -0.216196
                                      0.118788      -0.040339
       1       3    3py               4.336251      -0.783228
                                      1.288182      -0.542955
                                      0.403776      -0.216196
                                      0.118788      -0.040339
       1       3    3pz               4.336251      -0.783228
                                      1.288182      -0.542955
                                      0.403776      -0.216196
                                      0.118788      -0.040339

       1       4    4px               4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       0.099404
       1       4    4py               4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       0.099404
       1       4    4pz               4.336251       0.000000
                                      1.288182       0.000000
                                      0.403776       0.000000
                                      0.118788       0.099404

       2       1    3dx2              0.600000       0.673247
       2       1    3dxy              0.600000       1.166097
       2       1    3dxz              0.600000       1.166097
       2       1    3dy2              0.600000       0.673247
       2       1    3dyz              0.600000       1.166097
       2       1    3dz2              0.600000       0.673247


   - Potential information

     Potential name: GTH

     Description: Goedecker-Teter-Hutter pseudopotential
                  - Goedecker, Teter, Hutter, PRB 54, 1703 (1996)
                  - Hartwigsen, Goedecker, Hutter, PRB 58, 3641 (1998)

     Gaussian exponent of the core charge distribution:     4.109048

     Electronic configuration (s p d ...):  2  2

     Parameters of the local part of the GTH pseudopotential:

             rloc        C1          C2          C3          C4
           0.348830   -8.513771    1.228432

     Parameters of the non-local part of the GTH pseudopotential:

      l      r(l)      h(i,j,l)

      0    0.304553    9.522842
      1    0.232677


 TOTAL NUMBERS AND MAXIMUM NUMBERS

  Total number of

  - Atomic kinds:                       1
  - Atoms:                              1
  - Shell sets:                         2
  - Shells:                             5
  - Primitive Cartesian functions:      5
  - Cartesian basis functions:         14
  - Spherical basis functions:         13

  Maximum angular momentum quantum number of the

  - Orbital basis functions:                        2
  - Local part of the GTH pseudopotential:          2
  - Non-local part of the GTH pseudopotential:      0


 ATOMIC COORDINATES IN ANGSTROM

  Atom  Kind  Element       X           Y           Z       Z(eff)    Mass

     1    1   C     6    0.000000    0.000000    0.000000      4    12.0112


 SCF PARAMETERS

  Density guess: ATOMIC

  charge:                0
  nrebuild:              1
  max_scf:              30
  max_diis:              4
  eps_scf:            1.00E-05
  eps_diis:           1.00E-01
  eps_eigval:         1.00E-05
  p_mix:              0.40
  work_syevx:         1.00
  level_shift [a.u.]: 0.00
  smear [a.u.]:       0.02


 SCF WAVEFUNCTION OPTIMIZATION

  Number of electrons:                             4
  Number of occupied orbitals:                     2
  Number of orbital functions:                    13
  Number of independent orbital functions:        13

  Step  Update method            Time           Convergence        Total energy
  -----------------------------------------------------------------------------
     1  Mixing  0.40             5.85          0.0000160864       -5.3386344470
     2  Mixing  0.40             6.05          0.0000062458       -5.3386344472

  *** SCF run converged ***


  Total electronic density (r-space):         -3.9999999858        0.0000000142
  Total core charge density (r-space):         4.0000000000       -0.0000000000
  Total charge density (r-space):                                  0.0000000142
  Total charge density (g-space):                                  0.0000000142

  Overlap energy of the core charge distribution:                  0.0000000000
  Self energy of the core charge distribution:                   -12.9389984974
  Core Hamiltonian energy:                                         3.5052611006
  Hartree energy:                                                  5.4672736365
  Exchange-correlation energy:                                    -1.3721706869
  Total energy:                                                   -5.3386344472


 MO EIGENVALUES AND MO OCCUPATION NUMBERS

                                1            2            3            4
                            -0.495651    -0.193625    -0.193625    -0.193625

                             2.000000     0.666667     0.666667     0.666667


                                5            6            7            8
                             0.300295     0.300295     0.300295     0.429831

                             0.000000     0.000000     0.000000     0.000000


                                9           10           11           12
                             1.185245     1.185245     1.185245     1.185245

                             0.000000     0.000000     0.000000     0.000000


                               13
                             1.185245

                             0.000000




 -------------------------------------------------------------------------------
 -                                                                             -
 -                                T I M I N G                                  -
 -                                                                             -
 -------------------------------------------------------------------------------
 SUBROUTINE                        CALLS  CPU TIME T  PERFORMANCE   ELAPSED TIME
 CP2K                                  1     13.94 I       ---            15.19
 Quickstep                             1     13.94 I       ---            15.19
 scf                                   1     13.84 I       ---            15.07
 build_kohn_sham_matrix                2     11.90 I       ---            12.95
 fft_wrap_pw1pw2                       8      9.18 I       ---             9.81
 calculate_rho_elec                    2      4.78 I       ---             5.62
 init_grid                             1      1.85 I       ---             2.01
 calculate_xc_potential                2      1.28 I       ---             1.28
 PW_SCATTER                            4      0.94 I   1.6 Mcopy           0.94
 PW_GATHER                             4      0.31 I   4.7 Mcopy           0.35
 -------------------------------------------------------------------------------


  **** **** ******  **  PROGRAM ENDED AT                Thu Apr 18 10:35:31 2002
 ***** ** ***  *** **   PROGRAM RAN ON                                    lilith
 **    ****   ******    PROGRAM RAN BY                                     krack
 ***** **    ** ** **   PROGRAM PROCESS ID                                105472
  **** **  *******  **  PROGRAM RAN IN       /gpfs/home/krack/work/cp2k/tests/QS
